support-frontend/assets/helpers/subscriptionsForms/formReducer.js

```diff
@@ -43,9 +43,9 @@ function createFormReducer(
     billingAddressIsSame: null,
     billingPeriod: initialBillingPeriod,
     titleGiftRecipient: null,
-    firstNameGiftRecipient: '',
-    lastNameGiftRecipient: '',
-    emailGiftRecipient: '',
+    firstNameGiftRecipient: null,
+    lastNameGiftRecipient: null,
+    emailGiftRecipient: null,
```

Comment:
shoudl all the "gift" stuff be bound together more tightly to avoid so many codependent nullable vaiables?  Like a `GiftInfo` data type that is a sum type containing either Gift(title, firstname, lastname, email) or NonGift

Reply:
Yes I think it should and it is in the Scala model, but it had already been implemented like this when I got to it and it would be quite a big change to refactor it so I left it as is.